lib/packages.cf

```diff
@@ -1003,7 +1003,7 @@ body package_method yum
       package_list_arch_regex    => "$(rpm_knowledge.rpm3_arch_regex)";
 
       package_installed_regex => ".*";
-      package_name_convention => "$(name)-$(version).$(arch)";
+      package_name_convention => "$(name)";
```

Comment:
Wouldn't this result in being unable to specify the version and/or architecture for a package?

Reply:
Likely a better fix would be to change the C code to anchor the `$(version).$(arch)` portion to the end of the name. I can investigate that instead.